tests/test_redmine.py

```diff
@@ -41,40 +43,91 @@ def fail_line_matrix():
     return lines
 
 
+multiple_ticket_lines = [
+    'issues #1 #2 #3',
```

Comment:
this is not testing `issue`, `ticket`, `tickets`, `redmine`, `redmines`, `bug` and `bugs` (maybe it wasn't before but it is trivial to add them)